src/python/turicreate/test/test_models.py

```diff
@@ -6,6 +6,9 @@
 from __future__ import print_function as _
 from __future__ import division as _
 from __future__ import absolute_import as _
+
+# mark it as part of toolkits
+import turicreate.toolkits as _  # noqa
```

Comment:
So I'm guessing the changes in this file and the changes in `test_text_analytics.py` where made so that those test would be skipped when testing the minimal wheel. Is that right? If so, I think it illustrate why we probably want a different approach.

Reply:
agree.

Reply:
https://github.com/apple/turicreate/pull/3171